e2e/windows/README.md

```diff
@@ -78,7 +78,7 @@ windows 23H2 gen1 and windows 23H2 gen2.
 ## Generate and Update current windows test images
 
 Currently, we use the latest test images ""
-/subscriptions/$AZURE_BUILD_SUBSCRIPTION_ID/resourceGroups/$AZURE_BUILD_RESOURCE_GROUP_NAME/providers/Microsoft.Compute/galleries/$AZURE_BUILD_GALLERY_NAME/images/windows-e2e-test-$WINDOWS_E2E_IMAGE/versions/latest" (
+/subscriptions/$AZURE_E2E_IMAGE_SUBSCRIPTION_ID/resourceGroups/$AZURE_BUILD_RESOURCE_GROUP_NAME/providers/Microsoft.Compute/galleries/$AZURE_BUILD_GALLERY_NAME/images/windows-e2e-test-$WINDOWS_E2E_IMAGE/versions/latest" (
```

Comment:
The image definition name does not contain `e2e-test-`

Reply:
Thanks. Updated.